internal/store/installation.go

```diff
@@ -197,15 +197,15 @@ func (sqlStore *SQLStore) CreateInstallation(installation *model.Installation) e
 		SetMap(map[string]interface{}{
 			"ID":               installation.ID,
 			"OwnerID":          installation.OwnerID,
+			"GroupID":          installation.GroupID,
+			"GroupSequence":    nil,
```

Comment:
Just double-check that we want `nil` here and not an `int` value since `SetMap` takes a `map[string]interface{}` so the compiler won't enforce types signatures here. It's not obvious to me which would be correct and it'd be an easy mistake to make -- feel free to just resolve this comment once you read it if this is as-intended :)